libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -189,6 +196,8 @@ class AP_UAVCAN : public AP_HAL::CANProtocol {
     } _led_conf;
 
     HAL_Semaphore _led_out_sem;
+
+    AP_InertialSensor::DeltaAccumulator _imu_accumulators[INS_MAX_INSTANCES];
```

Comment:
I think we could dynamically allocate this array. Most people won't turn on this feature, so why use the memory?

Reply:
Is the 24 bytes enough to care about?